apim/3.x/templates/api/api-deployment.yaml

```diff
@@ -110,11 +110,11 @@ spec:
             - name: {{ .Values.api.service.internalPortName }}
               containerPort: {{ .Values.api.service.internalPort }}
             {{- if .Values.api.http.services.core.http.enabled }}
-            - name: {{ .Values.api.name }}-techapi
+            - name: {{- printf "%s-%s" .Values.api.name "techapi" | trunc 15 }}
```

Comment:
```suggestion
            - name: {{ printf "%s-%s" .Values.api.name "bridge" | trunc 15 | trimSuffix "-" }}
```
seems to work better otherwise the test not work

📝 We have some note [here](https://github.com/gravitee-io/helm-charts/blob/master/apim/3.x/TESTS.md) to run test

Reply:
if we add a trim here like this. it is possible that the ports have the same name if the api name is long. As this will remove the suffix `techapi`

WDYT about something like this :
`{{ printf "%s-%s" (.Values.api.name | trunc 8 | trimSuffix "-") "techapi" }}`

Reply:
Sounds good to me.
I will update the code and push branch.